lms/djangoapps/certificates/migrations/0009_auto_20170912_1425.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
```

Comment:
please give this file a better name than `auto_date_number`.  People look at these file names, especially when things break and so they should be clear and useful.